src/terra-dev-site/about/ComponentStandards.b.about.mdx

```diff
@@ -45,21 +47,21 @@ Terra components are built following the mobile-first approach and utilize the f
 
 ## Mobile Support
 
-Components should:
+Components must:
 * Support gestures when possible (swiping / pinching)
-* Not use hover events
+* Not use hover events without an alternative method of accessing the same information
 * Avoid small tap target sizes
 
 ## Accessibility (A11y)
 
-We should strive to make components accessible. Use the following guides to help build accessible components.
+Terra components comply to WCAG 2.0 standards. Use the following guides to help build accessible components.
 * [WCAG 2.0](https://www.w3.org/TR/WCAG20/)
```

Comment:
should we specify [all of these](https://github.com/cerner/terra-toolkit-boneyard/blob/master/src/wdio/services/TerraCommands/axe-command.js#L51) or are these descriptions inclusive, where-as axe is not?

Reply:
I think it would be sufficient to say that we comply at the AAA level. WCAG is inclusive, it's just axe that is additive (which is weird).